mappings/net/minecraft/item/EnchantedBookItem.mapping

```diff
@@ -8,5 +8,13 @@ CLASS net/minecraft/class_1428 net/minecraft/item/EnchantedBookItem
 		ARG 2 list
 	METHOD method_4609 getAsItemStack (Lnet/minecraft/class_1134;)Lnet/minecraft/class_1071;
 		ARG 1 info
+	METHOD method_4610 getLootTable (Ljava/util/Random;)Lnet/minecraft/class_847;
+		ARG 1 random
 	METHOD method_4611 getEnchantmentNbt (Lnet/minecraft/class_1071;)Lnet/minecraft/class_474;
 		ARG 0 stack
+		ARG 1 itemStack
+	METHOD method_6347 getLootTable (Ljava/util/Random;III)Lnet/minecraft/class_847;
+		ARG 1 random
+		ARG 2 minCount
```

Comment:
i think it would be better to use “min” and “max” here

Reply:
Updated with 93c6a0f5ab31b0ced83a6d68e720b27c78c4065a